File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/mesh/persistence/MeshExporter.java
/**
 * 
 */
package com.abubusoft.xenon.mesh.persistence;

import java.io.File;

import com.abubusoft.xenon.mesh.Mesh;
import com.abubusoft.xenon.mesh.persistence.wavefront.WavefrontExporter;

/**
 * @author Francesco Benincasa
 *
 */
public abstract class MeshExporter {

	public static boolean exportWavefront(File file, Mesh mesh)
	{
		return WavefrontExporter.export(file, mesh);
	}
}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/mesh/persistence/MeshExporter.kt
/**
 *
 */
package com.abubusoft.xenon.mesh.persistence

import com.abubusoft.xenon.mesh.Mesh
import com.abubusoft.xenon.mesh.persistence.wavefront.WavefrontExporter
import java.io.File

/**
 * @author Francesco Benincasa
 */
object MeshExporter {
    fun exportWavefront(file: File?, mesh: Mesh?): Boolean {
        return WavefrontExporter.export(file, mesh!!)
    }
}